packages/terra-overlay/tests/wdio/overlay-spec.js

```diff
@@ -1,6 +1,163 @@
-/* global browser, Terra */
+/* global before, browser, Terra */
 
 describe('Overlay', () => {
+  describe('Default', () => {
+    beforeEach(() => browser.url('/#/tests/overlay-tests/default'));
+
+    Terra.should.beAccessible({ selector: '#default-overlay' });
+    Terra.should.matchScreenshot({ selector: '#default-overlay' });
+
+    it('Expect default to have full screen', () => {
+      expect(browser.getAttribute('#default-overlay', 'class')).contains('fullscreen');
```

Comment:
I would say if we have screen shots depicting things like full screen etc, we do not have to write explicit wdio tests for it. If it is something that cannot be captured in a screenshot like spin animation/scrollability then add tests for it.